### DISCUSSION TITLE
Release Plan for Instruction Templates Used in Evaluation?

### DISCUSSION THREAD

**Discussion by ThreeCirclesK** [ibm-granite/discussions#2] [Q&A] [answered]
Hello!

I'm impressed with the promising results your model offers.

Are there any plans to release  **specific instruction templates** utilized for each instruction-tuned model in your experiments?

Providing these templates would greatly assist researchers.

**Comment by rpand002** [comment, marked as answer]
For each instruction-tuned model, we used the specific prompt format mentioned in their HuggingFace model card to obtain the best results (as suggested by the corresponding model creators). We have added the following few sentences in Section 6.1.1 of our technical paper. In addition to make it clear, we will also add/release all the prompt templates used for evaluation in our GitHub repo very soon.

`In constructing prompts for instruction-tuned models, we adhere to the formats provided in their official examples. We search for a suitable prompt format in the HuggingFace model card, GitHub repository, and formal publications or technical reports. `
